src/app/containers/Include/index.jsx

```diff
@@ -15,7 +18,7 @@ const componentsToRender = {
   },
   canonical: {
     idt1: props => <Canonical {...props} />,
-    idt2: props => <Canonical {...props} />,
+    idt2: props => <Idt2Canonical {...props} />,
```

Comment:
Is there now an issue to re-instate the coverage?

Reply:
What coverage has been lost @rhenshaw56 ?

Reply:
I think @rhenshaw56 is referring to the adjustment in `containers/Include/index.test.js`. Coverage has been maintained, but following the refactor, we will probably need to look at tidying up the Include container unit tests.

Reply:
Just an assertion to check that we're using the right import for each include type, we could probably do it here but that will require a slight refactor of the tests.